src/parser/rogue/assassination/modules/Abilities.js

```diff
@@ -267,6 +263,42 @@ class Abilities extends CoreAbilities {
         category: Abilities.SPELL_CATEGORIES.UTILITY,
         // While this actually has a 0.5s CD, it shows up weird in the Abilities tab if we set that
       },
+      // Covenant Abilities
+      {
+        spell: SPELLS.SEPSIS,
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        gcd: {
+          static: 1000,
```

Comment:
are these actually static GCDs? or can they be reduced by haste?

if they can be reduced by haste then base: 1000